cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -413,12 +413,13 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 
 	volumeController := persistentvolumecontroller.NewPersistentVolumeController(
 		clientset.NewForConfigOrDie(restclient.AddUserAgent(kubeconfig, "persistent-volume-binder")),
+		sharedInformers.PersistentVolumes().Informer(),
 		s.PVClaimBinderSyncPeriod.Duration,
 		provisioner,
 		ProbeRecyclableVolumePlugins(s.VolumeConfiguration),
 		cloud,
 		s.ClusterName,
-		nil, nil, nil,
+		nil, nil, /* claimSource, eventRecorder */
```

Comment:
Similar changes are needed in `/contrib/mesos/pkg/controllermanager/controllermanager.go`, otherwise some unit test won't compile.